interpreter/interpreter_test.go

```diff
@@ -39,7 +39,8 @@ import (
 type testCase struct {
 	name string
 	E    string
-	I    map[string]interface{}
+	Env  []*exprpb.Decl
+	I    interface{}
```

Comment:
All uses below still assume it's a `map[string]interface{}`. Just wanted to double check you'd like to lessen the restriction here.

Reply:
`EmptyActivation()` isn't actually a `map[string]interface{}` in some of the test cases below, so the type-widening was intentional.